docs/release-notes/2.0.0.md

```diff
@@ -0,0 +1,69 @@
+# Version 2.0.0 (2023-02-)
+
+| Release date |   |
+| ------------ | ------------------------------------------ | 
+| Installation | [Install `pg_stat_monitor`](../install.md) | 
+
+## Release highlights
+
+* `pg_stat_monitor` is Generally Available, enabling you to use it in production environments.
+* Added support of PostgreSQL 15:
+
+   | Data type changes | Columns added | Columns renamed | 
+   | ----------------- | ------------- | --------------- |
+   | `userid` - oid <br>  `queryid` - bigint <br> `top_queryid` - bigint <br> | `dbid` <br> `user` <br> `temp_blk_read_time` <br> `temp_blk_write_time` <br> `jit_functions` <br> `jit_generation_time` <br> `jit_inlining_count` <br> `jit_inlining_time` <br> `jit_optimization_count` <br> `jit_optimization_time` <br> `jit_emission_count` <br> `jit_emission_time` | `rows_retrieved` to `rows` <br> `plans_calls` to `calls` | 
+
+* Improved consistency with `pg_stat_statements` - Now `pg_stat_monitor` view has exactly the same column names, columns and data types as the `pg_stat_statements` view for every major version of PostgreSQL from 11 till 15. 
+* Histogram output now allows users to see the whole data range, including the queries with the execution time lower than the minimum time, and exceeding the maximum time specified by the user. 
+* Now the query text is stored in memory, improving the memory buffer management and preventing its overflow
+* CPU and user time are now captured for utility statements, helping you understand the resource consumption for these statements execution. 
+* [PMM](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) integration improvements:
+    * An indication of buckets as "done" vs "current" eliminates the need for the tool to evaluate the bucket status and facilitates the accurate data display 
+    * The generation of a unique ID of a query, independent from PostgreSQL versions, database, users or schemas, improves observability of query planning, execution and performance
+
+### Incompatibility with previous releases:
+
+* The `pg_stat_monitor_settings` is deprecated and removed. To retrieve the configuration parameters, use `pg_settings`.
```

Comment:
The `pg_stat_monitor_settings` what`? Variable?

Reply:
Fixed